src/main/java/org/rumbledb/runtime/quantifiers/QuantifiedExpressionVarIterator.java

```diff
@@ -57,7 +58,7 @@ public QuantifiedExpressionVarIterator(
         // this.sequenceType = sequenceType;
```

Comment:
Just noticed this commented out line. Is this on our TODO list ?

Reply:
Yes. It's commented out because it's not used (yet) and so avoids a Java warning. But in the future, quantified expression type checking should be implemented.